### DISCUSSION TITLE
Import pocketbase records

### DISCUSSION THREAD

**Discussion by jramke** [pockethost/pockethost#263] [Q&A]
Hey is it possible to import the records?

via the pocketbase ui you can only import the collections.

I also tried to upload the pb_data folder via ftp but now im getting a "attempt to write to read only database".

**Comment by benallfree** [comment]
>  is it possible to import the records?

This sounds like a PocketBase question, have you tried posting there?

> im getting a "attempt to write to read only database"

This might be an issue with PocketHost, I'll need to look into it.

**Comment by jramke** [comment]
Thanks for the quick answer.
Maybe i missed something on just trying to upload the folder, im pretty new to this server hosting ans database shit
